examples/src/index.js

```diff
@@ -53,7 +53,6 @@ const AsyncDemo = () => {
       <HCaptcha
         ref={captchaRef}
         sitekey="917ba1eb-0b37-486e-9c90-39f3cb7b2579"
```

Comment:
Needs to be changed to a [test sitekey](https://docs.hcaptcha.com/#test-key-set-publisher-account).